Launcher/Classes/PlistOperations.swift

```diff
@@ -9,6 +9,23 @@ class PlistOperations {
         dictionaryKey = key
         let documentDirectory = NSSearchPathForDirectoriesInDomains(.cachesDirectory, .userDomainMask, true)[0] as String
         plistPath = documentDirectory.appending(path)
+        
+        if let integratedSettingsPath = Bundle.main.path(forResource: "CalabashLauncherSettings", ofType: "plist"), !fileManager.fileExists(atPath: plistPath){
+            self.copyFile(atPath: integratedSettingsPath, toPath: plistPath)
+        }
+    }
+    
+    func copyFile(atPath: String, toPath: String) {
+        do {
+            try fileManager.copyItem(atPath: atPath, toPath: toPath)
+        }
+        catch let error as NSError {
+            print("Ooops! Something went wrong: \(error)")
```

Comment:
I’d add a bit more information here so it is clear that the file manager error is related to copying a file while handling some plist.

Reply:
caught `(error)` is quite verbose ;)

Reply:
Why do you convert to an `NSError` as well? A simple `catch` ought to be enough.